inc/saitunnel.h

```diff
@@ -323,8 +323,30 @@ typedef enum _sai_tunnel_type_t
 
     SAI_TUNNEL_TYPE_MPLS,
 
+    SAI_TUNNEL_TYPE_IPINIP_ESP,
```

Comment:
this tunnel types are relevant to ipsec
what's the use of introducing to 1.7 if we don't bring the full ipsec feature

Reply:
perhaps it's better to clean up
    SAI_TUNNEL_ATTR_SA_INDEX,
    SAI_TUNNEL_ATTR_IPSEC_SA_PORT_LIST,
that come from https://github.com/opencomputeproject/SAI/commit/231e67e4fb627265f0d583ad3be88271395e247a
as we don't have ipsec in 1.7
then we don't need the new _ESP types

Reply:
thank you!
Clean up the ipsec related interfaces and attributes.

Test with `make -C ./meta/`